### ISSUE TITLE
A admin user needs app to do majority of the work of digging up relevant info from Twitter

### ISSUE THREAD

**Issue by cczona** [callbackwomen/callback-women-rails#18]
tl;dr: Finding relevant tweets is a clunky, time-consuming, pain in the butt. Stuff needs to be collected far better, continuously, and cached.
### The most common types of Twitter searches currently:
1. Seeking additional conferences to follow, via searching Twitter as a whole
2. Seeking CFPs to share, via searching CallbackWomen's timeline for keywords (e.g. `"CFP"`, `"call for proposals"`, etc).
3. Seeking, in various ways, other relevant info to share. e.g. travel funding, diversity scholarships, speaker mentorship, childcare, codes of conduct, etc.
### Good news:
1. Twitter search supports booleans and search criteria such as date range, username, etc. `http://thesocialchic.com/2013/04/26/how-to-master-twitter-search/` Sweet!
### Pain points:
1. Clients implement Twitter boolean searches poorly, if at all. Support for search by criteria is rarer.
2. Twitter's API doesn't support regex `http://stackoverflow.com/questions/23363940/using-regular-expression-in-twitter-api`
3. There aren't (afaik) clients that let you run regexes against result sets it has obtained via the API
4. Some conferences call things a CFP that don't meet CallbackWomen's particular criteria.
5. Some conferences use less particular vocabulary (for instance, calls in languages other than English; or peculiaries such as "C4P")
6. It takes a lot of searches, separately, to find the genuine hits. 
7. Because the searches have to be separate and imprecise, there's a lot of noise to wade through each time
8. There are other relevant tweets to cull, such as regarding scholarships, childcare, mentorship, etc. These also have a range of expressions to match.
9. Stuff rolls of of timelines too fast. If you're not checking search results daily, relevant tweets get missed. Which is a problem since some conferences only tweet a CFP announcement once ("open!") or twice ("deadline is tonight!"). So not checking frequently can cause a CFP to be missed altogether.
### Proposed solution:

Ongoing administrative searches of tweets. A collection of complex searches, whose results are  merged/de-duped/refined by the app, and cached by the app so that admin(s) can pull from that narrower resultset when convenient. 

Setting administrative searches via an .ini file is fine. There probably needs to be some kind of basic sanity checking before passing searches forward to the API.
### Alternative solutions:

I'm open to alterative ideas for dealing with the core problem. Thoughts?
